lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -145,9 +145,10 @@ def answer_label
       def answer_labels
         Array.wrap(@current["value"]).map do |answer_idx|
           begin
-            answer_string = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]['label']
+            task_answer = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]
```

Comment:
Agreed that there needs to be something in the invalid label's place for researchers.